### PR TITLE
See if we're missing IE11 data due to ajax requests

### PR DESCRIPTION
**Pull request by markstephens**

### CODE REVIEW

File: src/javascript/core/send.js [hunk 1]
```diff
@@ -99,6 +99,19 @@ function sendRequest(request, callback) {
 	if (!(settings.get('developer') && settings.get('no_send'))) {
 		transport.send(url, stringifiedData);
 	}
+
+	// Detect IE11
+	const ie11 = !!navigator.userAgent.match(/Trident\/7\./);
+	if (ie11 && request.category === 'page' && request.action === 'view') {
```
Comment (wheresrhys) [inline src/javascript/core/send.js]: Isn't there a danger of other systems that hang off spoor getting double page views?

**Comment by rowanbeentje** [comment]
Hmm.  This is within the javascript path of o-tracking - so may not catch all levels of errors.  It may be better to use a tracking pixel directly on certain types of pages, so we don't need any of the o-tracking dependencies - this is really only isolating the ajax transport part...

File: src/javascript/core/send.js (hunk 1 above)
Comment (markstephens) [reply src/javascript/core/send.js]: No, I'm rewriting the category below before resending using the image transport

**Comment by markstephens** [comment]
@rowanbeentje as mentioned in slack, we have seen reports of ajax errors in ie11

**Comment by rowanbeentje** [comment]
@markstephens Ah ok - if we know it's an ajax error, how will this PR move things forward? Or is it just to verify that theory?

**Comment by markstephens** [comment]
@rowanbeentje This is to test the theory, then we'll need something else later on to always stick to Images for IE11 (or whatever the fix is)

File: src/javascript/core/send.js [hunk 2]
```diff
@@ -99,6 +99,19 @@ function sendRequest(request, callback) {
 	if (!(settings.get('developer') && settings.get('no_send'))) {
 		transport.send(url, stringifiedData);
 	}
+
+	// Detect IE11
+	const ie11 = !!navigator.userAgent.match(/Trident\/7\./);
```
[the remark this answers is not in the document]
Comment (markstephens) [reply src/javascript/core/send.js]: Got a suggestion of the code? It already took me a bunch of googling to get that line!

[the remark this answers is not in the document]
Comment (rowanbeentje) [reply src/javascript/core/send.js]: `!!window.MSInputMethodContext && !!document.documentMode` works in both Standards and Compatibility mode (wow, compatibility is a bit broken)

**Comment by rowanbeentje** [comment]
(Added https://github.com/Financial-Times/o-tracking/issues/187 to revert this)

**Comment by markstephens** [comment]
@rowanbeentje :) Thank you!!

Do you think it looks decent? The tests seem to pass on Circle, but that's all I've got to go on....

**Comment by markstephens** [comment]
Thank you @rowanbeentje 

@chrisbrownuk has been comparing page view events to memb access logs, he can adjust his query to include these ie11 events and see if we are capturing users' usage we weren't capturing before.
There is already a 1:1 mapping between them as they have the same event root ID.
I'm cautious of moving to before the original send for fear of breaking something!!

**Comment by markstephens** [comment]
I'm going to merge!

**Pull request merged**